pkg/lib/config/hook.go

```diff
@@ -7,7 +7,6 @@ var _ = Schema.Add("HookConfig", `
 	"properties": {
 		"sync_hook_timeout_seconds": { "$ref": "#/$defs/DurationSeconds" },
 		"sync_hook_total_timeout_seconds": { "$ref": "#/$defs/DurationSeconds" },
-		"handlers": { "type": "array", "items": { "$ref": "#/$defs/HookHandlerConfig" } },
```

Comment:
If we remove the config, then do we need to provide migration for existing production apps? Otherwise once we upgrade the server image tag, the existing app will have config error.

Reply:
I thought we will update the config for the specific project manually, or you think using script is better? I can add the script

Reply:
Either way has downtime. We just have 1 app. So manually is fine.